src/index.html

```diff
@@ -9,6 +9,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="box">
+        <a href="#">
+          <img
+           src="images/logo.png"
+           alt="logo"
+           class="logo"
+          >
+        </a>
+        <ul class="list">
+          <li class="is-active">APPLE</li>
+          <li class="goods">SAMSUNG</li>
```

Comment:
1. it should also be links - wrap content into `a` tag (use anchor `#` in href)
2. instead of writing `SAMSUNG` in the upper case in HTML it's better to use styles for it
3. write `good` class - in single form for one item